database.js

```diff
@@ -16,7 +19,7 @@ const databaseConnect = async (config = appConfig) => {
 
 		}
 	} catch (error) {
-		console.log(`Something went wrong ${error}`);
+		logger.error('Something went wrong ', error);
```

Comment:
I think 'Something went wrong ' error message is not self-describing enough. May be makes sense to rename and provide better error message describing what exactly happened?